GraphicEditor/Repositories/MedicineManufacturerRepository.cs

```diff
@@ -0,0 +1,60 @@
+﻿using GraphicEditor.Repositories.Interfaces;
+using Model.Hospital;
+using MySql.Data.MySqlClient;
+using System;
+using System.Collections.Generic;
+
+namespace GraphicEditor.Repositories
+{
+    public class MedicineManufacturerRepository : IMedicineManufacturerRepository
+    {
+        private MySqlConnection connection;
+        public MedicineManufacturerRepository()
+        {
+            connection = new MySqlConnection("server=localhost;port=3306;database=mydb;user=root;password=root");
+        }
+
+        private List<MedicineManufacturer> GetMedicineManufacturers(String query)
+        {
+            connection.Close();
```

Comment:
Why is this everywhere?

Reply:
We are calling other repositories in the time of the transactions.